### PR TITLE
lkp-exec/install: add OS version check and mirror source for centos

### PR DESCRIPTION
**Pull request by jiaqizhao122**
fix for the following error:
[root@localhost lkp-tests]# lkp install
Use: /root/lkp-tests/distro/installer/centos install bc gawk gzip kmod numactl rsync time which automake bison bsdtar bzip2 ca-certificates cpio fakeroot flex gcc git glibc-devel glibc-static glibc-devel.i686 glibc-static.i686 libtool make make automake gcc gcc-c++ kernel-devel rpm-build openssl openssl-devel patch perf perl-IPC-Run rsync ruby ruby-devel wget ......
软件包 git-2.39.3-1.el9.x86_64 已安装。
软件包 glibc-devel-2.34-95.el9.x86_64 已安装。
未找到匹配的参数: glibc-static
未找到匹配的参数: glibc-static.i686
软件包 libtool-2.4.6-45.el9.x86_64 已安装。
......
软件包 openssl-1:3.0.7-25.el9.x86_64 已安装。
未找到匹配的参数: perl-IPC-Run
软件包 wget-1.21.1-7.el9.x86_64 已安装。
错误：没有任何匹配: glibc-static glibc-static.i686 perl-IPC-Run Cannot install some packages of lkp depends

### CODE REVIEW

File: lkp-exec/install [hunk 1]
```diff
@@ -217,6 +217,15 @@ install_packages() {
 		fi
 	fi
 
+	if [ "$distro" = "centos" ]; then
+		version=$(cat /etc/os-release | grep "^VERSION=" | cut -d '"' -f 2)
```
Comment (rli9) [inline lkp-exec/install:221]: use local version

Comment (jiaqizhao122) [reply lkp-exec/install:221]: ok

File: lkp-exec/install [hunk 2]
```diff
@@ -351,11 +351,29 @@ install_base_support()
 	create_host_config
 }
 
+enable_centos_source(){
+    if ! yum list installed | grep -q yum-utils; then
+        yum install yum-utils -y
+    fi
```
Comment (rli9) [inline lkp-exec/install:357]: add empty line after fi

File: lkp-exec/install [hunk 3]
```diff
@@ -351,11 +351,29 @@ install_base_support()
...
+    if [ "$_system_version" = 9 ]; then
+        dnf config-manager --set-enabled crb
+        echo "open source crb successfully"
```
Comment (rli9) [inline lkp-exec/install:360]: move echo before dnf, as echo "enable source crb"

File: lkp-exec/install [hunk 4]
```diff
@@ -351,11 +351,29 @@ install_base_support()
...
+    elif [ "$_system_version" = 8 ]; then
+        dnf config-manager --set-enabled powertools
+        echo "open source powertools successfully"
```
Comment (rli9) [inline lkp-exec/install:363]: move echo before dnf

File: lkp-exec/install (hunk 2 above)
Comment (jiaqizhao122) [reply lkp-exec/install:357]: ok

File: lkp-exec/install (hunk 3 above)
Comment (jiaqizhao122) [reply lkp-exec/install:360]: ok

File: lkp-exec/install (hunk 4 above)
Comment (jiaqizhao122) [reply lkp-exec/install:363]: ok

**Pull request closed without merging**